Sources/SpeziLicense/Views/PackageCell.swift

```diff
@@ -0,0 +1,74 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftPackageList
+import SwiftUI
+
+
+struct PackageCell: View {
+    let package: Package
+    
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading, spacing: 4) {
+                Text(package.name).font(.headline)
+                HStack {
+                    Text(getPackageDetails(package: package))
+                        .font(.caption)
+                    if let licenseType = package.getLicenseType(license: package.license) {
+                        Text(licenseType.spdxIdentifier)
+                            .font(.caption)
+                            .fontWeight(.semibold)
+                            .padding(2)
+                            .background(Color(.systemGray5))
+                            .cornerRadius(4)
+                    }
+                }
+            }
+            Spacer()
+            Button(action: {
+                UIApplication.shared.open(package.repositoryURL)
+            }) {
+                Image(systemName: "safari.fill")
+                    .imageScale(.large)
+            }.buttonStyle(PlainButtonStyle())
```

Comment:
I do generally prefer the following code formatting to make it a bit clearer where brackets start and end:
```suggestion
            Button(
                action: {
                    UIApplication.shared.open(package.repositoryURL)
                },
                label: {
                     Image(systemName: "safari.fill")
                        .imageScale(.large)
                }
            )
                .buttonStyle(PlainButtonStyle())
```